### PR TITLE
Welcome Microverse Project

### PR DESCRIPTION
**Pull request by SaveryIV**
Second Review @NahnahAJ :

    **I already make these changes:**

     - Set up the linters for HTML and CSS.
     - _test.md_ file is now in .gitignore.
     - The folder _future-features_ is now deleated.
     - MIT license in the Main branch.

### CODE REVIEW

**Review by nahnahdev — CHANGES_REQUESTED** [review]
## Project Status: CHANGES REQUIRED :recycle: 
Hi @SaveryIV  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Good commit messages :heavy_check_mark: 
- All relevant files created :heavy_check_mark: 

## Required Changes ♻️
- [ ] Great job so far @SaveryIV there are just a few changes you need to make and you'll be on your way to the next project :wink: 
- [ ] Please ensure that your branch contains a short descriptive summary of the changes that were made in the current pull request and give your pull request title a more descriptive name. Instead of `Waiting for revision `, you can have something like `Hello Microverse Project`:100: 
image

- [ ] As mentioned in the [project requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md), you are to set up [linters for HTML and CSS](https://github.com/microverseinc/linters-config) but currently, you have no linters set up in your project so kindly do this as well :100: 
- [ ] I also noticed that you have included the  [Microverse Readme template](https://github.com/microverseinc/readme-template) but you didn't customize it to suit this project. This makes your project incomplete. Every section of the readme template is required with the exception of the FAQ section which is optional. 

- [ ] Follow the steps I have listed below to customize it to suit this project:

    • Remove the Microverse logo. If you have your logo, you can use it but I would advise you to remove it :100: 
 - Please apply the right project title instead of leaving the project title empty.
    • For the Tech Stack Section, Kindly remove the Server and Database subsections because the technologies listed there are not relevant to this project. For the Client section, kindly include HTML and CSS :+1: 

    • For the Key Features section, kindly highlight what this project is all about. This will make it easy for those who go through this repository to know what was implemented by reading the README.md file. :wink: 

    • For the Prerequisite Section, kindly add tools that are needed before you can clone this repository on a local machine. Tools like:
        ◦ Installation of Git
        ◦ Installation of NPM
        ◦ A working browser
        ◦ Internet connection
        ◦ A code editor like Visual Studio Code
        ◦ Installation of Live Server plugging in Visual Studio Code
        ◦ Any other tool you can think about :100: 

    • For the Setup section, kindly add instructions for cloning your repository on a local machine :+1: 

    • For the Install section, kindly add instructions for setting up linters used in this project :+1: 

    • For the Usage section, include instructions for opening this project on a browser. For example, opening the project by launching the Live Server plugging in visual studio code or any other means that you know.

- For the Future Features section include 2 or more future features.

- Please include an MIT license file in your project. The link to your license should be included in your README file. The MIT License is a popular open-source license that allows others to use, modify, and distribute your code as long as they give credit to you the original author, and include the original license text. You can use this [link](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to learn how to add one to your project :wink: 

 - Kindly implement all the changes listed above. It will make your README.md file look professional and it will give you an idea of how to set README.md files for most of the projects you will execute as a Microverse student.  
- [x] Also, per the project requirements, `A test.md file with the text "This file should be ignored by git".` But currently, that file has been pushed to github. I will suggest that you ensure that that file is added to the `gitignore` file so it is ignored and not pushed to GitHub. :+1: 

image

- [x] Lastly, I noticed that you created this directory in your branch but this isn't required for this milestone so i will suggest that you delete it :+1: 
image


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by SaveryIV** [comment]
Hi! @NahnahAJ, now I have conflicts with the README, I don't know why, if you can help me with this pls

File: README.md [hunk 1]
```diff
@@ -283,6 +150,4 @@ I would like to thank...
 
 This project is [MIT](./LICENSE) licensed.
```
Comment (braincee) [inline README.md]: - [ ] Please I recommend adding an MIT licence file as suggest by the previous reviewer to your project. This is a provided [link](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to demonstrated how to add the file. 😄

**Review by braincee — CHANGES_REQUESTED** [review]
Hello @SaveryIV 👋 , I am @braincee your Code Reviewer

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


so-close

REQUIRED CHANGES ♻️
------

- [ ] Please I highly recommend adding linter files per project requirements as suggested by the previous reviewer. These files include `hintrc` and `stylelintrc.json`. 😄 
- [ ] Please I highly adding an author section to the README file. 😄 
- [ ] Please I recommend adding a "This file should be ignored by git" to the `test.md` file. 😄  


**_Please check the comments under the review._**

OPTIONAL CHANGES
--------

- [ ] [OPTIONAL] Please I recommend adding `html` and `css` files per the project requirement. Sorry It was not mentioned earlier by the previous reviewer. 😄

**_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @braincee in your question so I can receive the notification.

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._

----

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (SaveryIV) [reply README.md]: I alredy did it, now i have problems with the linters, what can i do? i follow all the steps. @braincee

**Review by souad988 — APPROVED** [review]
Hi @SaveryIV , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f6a4a5143327075565a70544d4f34765573302f67697068792e676966


## Highlights
✔No linter errors
✔Good PR title
✔Great efforts

## Optional suggestions
_Kindly don't forget to remove the default text under each section of your README as it was added to guide you in what content to put there and removing it makes your work clean and professional, also consider removing the FAQ section if not having a content for it as it is optional all other sections are required always keep them in your coming projects .
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**